src/resources/2025/BloodCubicZirconia.ts

```diff
@@ -0,0 +1,79 @@
+import { myBasestat, Skill, Stat } from "kolmafia";
+import { $item, $skill, $skills, $stat } from "../../index.js";
```

Comment:
```suggestion
import { $item, $skill, $skills, $stat } from "../../template-string.js";
```
never import from the index. vscode loves to do it tho